packages/performance/src/services/transport_service.ts

```diff
@@ -24,6 +24,14 @@ const INITIAL_SEND_TIME_DELAY_MS = 5.5 * 1000;
 const MAX_EVENT_COUNT_PER_REQUEST = 1000;
 const DEFAULT_REMAINING_TRIES = 3;
 
+// Most browsers have a max payload of 64KB for sendbeacon/keep alive payload.
+const MAX_SEND_BEACON_PAYLOAD_SIZE = 65536;
+// The max number of events to send during a flush. This number is kept low to since Chrome has a
+// shared payload limit for all sendBeacon calls in the same nav context.
+const MAX_FLUSH_SIZE = 40;
```

Comment:
How did we arrive at this 40? Is this heuristic driven or are we making it as a random pick. If a random pick, should we make it Remote COnfig configurable with default value of 40?

Reply:
I chose 40 since it seems that ~100 is the typical max we are able to send. I've made it a RC flag in case we need to modify it later.